CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * **master**
     * Added support for roaring imports which can speed up the import process by %30 for non-key column imports. This feature requires Pilosa on master branch.
+    * Removed import options.
```

Comment:
Let's change this to:
`* Removed import options related to timeout strategy.'
or something to that effect.

Reply:
I meant *import strategies* instead of *import options*. Updated